Woof/WoofTests/Model/KeyValueStorage/Extensions/KeyValueStorage+UnitTests.swift

```diff
@@ -4,6 +4,11 @@ extension KeyValueStorage {
         KeyValueStorage("Test-KeyValue-Storage")
     }
 
+    /// A storage for preferences
```

Comment:
Please be consistent with the existing doc comments. Smth like that:

```suggestion
    /// A test storage with user preferences to use in the unit tests.
```

Reply:
fixed